apps/meteor/client/views/admin/subscription/SubscriptionPage.tsx

```diff
@@ -48,15 +51,15 @@ const SubscriptionPage = () => {
 	const router = useRouter();
 	const { data: enterpriseData } = useIsEnterprise();
 	const { isRegistered } = useRegistrationStatus();
-	const { data: licensesData, isLoading: isLicenseLoading } = useLicense({ loadValues: true });
+	const { data: licensesData, isLoading: isLicenseLoading } = useLicenseWithCloudAnnouncement({ loadValues: true });
 	const syncLicenseUpdate = useWorkspaceSync();
 	const invalidateLicenseQuery = useInvalidateLicense();
 
 	const subscriptionSuccess = useSearchParameter('subscriptionSuccess');
 
 	const showSubscriptionCallout = useDebouncedValue(subscriptionSuccess || syncLicenseUpdate.isLoading, 10000);
 
-	const { license, limits, activeModules = [] } = licensesData || {};
+	const { license, limits, activeModules = [], trial } = licensesData?.license || {};
```

Comment:
why you use change this to `trial` nut ignored `cloudSyncAnnouncement` whit is the real focus here?

Reply:
We were testing some changes and this got left behind, although it didn't change the behavior, I agree this change should not be part of this PR.